model/clusters_mgmt/v1/add_on_install_mode_type.model

```diff
@@ -18,7 +18,7 @@ limitations under the License.
 enum AddOnInstallMode {
 	// This mode means that the the addon CRD exists in a single specific namespace.
 	// This namespace is reflected by the TargetNamespace addon field
-	SingleNamespace
+	OwnNamespace
```

Comment:
Is the comment still valid, or does that need updating?

Reply:
From my understanding its essentially the same thing, just with the new Sync Set change, MT only support Own and All namespaces. Own Namespace come from what is defined in target namespace